ckanext/justicehub_theme/templates/organization/stats.html

```diff
@@ -0,0 +1,25 @@
+{% extends "organization/read.html" %}
+
+{% block subtitle %}{{ _('Stats') }} - {{ c.group_dict.display_name }}{% endblock %}
+
+{% block page_primary_action %}
+{% endblock %}
+
+{% block primary_content_inner %}
+	<div class="row stats-box">
+            <div class="col-xs-6 col-sm-4">
+                <span class="stats">100{{ _('+') }}</span>
+                <span class="stats-text">{{ _('DOWNLOADS') }}</span>
+            </div>
+            <div class="col-xs-6 col-sm-4">
+                <span class="stats">200{{ _('+') }}</span>
+                <span class="stats-text">{{ _('Visits') }}</span>
+            </div>
+            <div class="col-xs-6 col-sm-4">
+                <span class="stats">{{ c.group_dict.num_followers }}{{ _('+') }}</span>
+                <span class="stats-text">{{ _('Followers') }}</span>
+            </div>
+        </div>
+    {% block page_pagination %}
+    {% endblock %}
+{% endblock %}
```

Comment:
can we have the real numbers instead of hardcoded ones in stats.

![image](https://user-images.githubusercontent.com/30413611/82806172-e0b3ed00-9ea2-11ea-9eb4-63f9952b528c.png)